src/common/src/catalog/mod.rs

```diff
@@ -46,13 +46,23 @@ pub type SourceVersionId = u64;
 /// The default version ID for a new source.
 pub const INITIAL_SOURCE_VERSION_ID: u64 = 0;
 
-pub const DEFAULT_DATABASE_NAME: &str = "dev";
+use std::env;
+
+pub static DEFAULT_DATABASE_NAME: LazyLock<String> =
```

Comment:
I'm wondering whether we should make them system parameters and allow users to override them with CLI args **on the first run**. Or it could be confusing. 🤔

Reply:
+1. BTW maybe it doesn't need to be a system param and just a CLI arg or sth, as it's only usage is init.

> No confusion since the user must have looked up for the variable and know exactly what he wants to do.

> The common usage is to specify in the docker file before starting the database.

IIUC, `POSTGRES_DB` is a feature of postgres docker image, instead of postgres kernel, which has other configuration mechanism for initialization.

So, I think this solution of course works, but it's hacky.

Reply:
+1. We need to make sure it only takes effect on the first run. After the database being initialized, the default db and username should be persisted.

I believe this is also the behavior of the `POSTGRES_DB`.